extensions/widgets/segmented/segmented.lcb

```diff
@@ -259,6 +260,8 @@ public handler OnLoad(in pProperties as Array)
 	put pProperties["showFrameBorder"]	into mShowFrameBorder
 	setSelected(pProperties["selectedSegment"], kDontPostMessage)
 
+	put pProperties["cornerRadius"] into mRadius
```

Comment:
@jameshale this needs to be guarded with `if "cornerRadius" is among the keys of pProperties then`